nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2022,7 +2032,7 @@ impl NexusExternalApi for NexusExternalApiImpl {
             .await
     }
 
-    async fn ip_pool_range_list(
+    async fn system_ip_pool_range_list(
```

Comment:
Note to self -- I think this should be a "normal", non-system endpoint. Users who can see an IP Pool should be able to list its ranges. But it does make the whole prefix / no-prefix thing more confusing.

Reply:
I talked a while ago with @ahl about this. I think we actually want to have a system and non-system endpoint for listing and viewing pools, and for listing ranges in a pool. The non-system versions are basically the same as the existing "project-scoped" implementations, showing the resources available for use in the silo of the calling user.